test/unit/gateways/merchant_e_solutions_test.rb

```diff
@@ -16,6 +16,8 @@ def setup
 
     @options = {
       order_id: '1',
+      recurring_pmt_num: 11,
+      recurring_pmt_count: 10,
```

Comment:
We should add some check in one of the unit test to make sure the value is added to the `post` body correctly.

Reply:
Done